third_party/amd/python/test/test_gluon_gfx1250.py

```diff
@@ -422,3 +422,189 @@ def test_runtime_tensor_copy(BLOCK_M, BLOCK_N):
+        scale_b = ttgl.load(b_scale_ptr)
+        scale_a = ttgl.convert_layout(scale_a, A_SCALE_LINEAR_LAYOUT)
+        scale_b = ttgl.convert_layout(scale_b, B_SCALE_LINEAR_LAYOUT)
+
+        a = ttgl.load(a_ptrs, mask=valid_k_a[None, :], other=0.0)
+        b = ttgl.load(b_ptrs, mask=valid_k_b[:, None], other=0.0)
+        a = ttgl.convert_layout(a, DOT_LAYOUT_A)
+        b = ttgl.convert_layout(b, DOT_LAYOUT_B)
+
+        accumulator = ttgl.amd.gfx1250.wmma_scaled(a, scale_a, DTYPE_A, b, scale_b, DTYPE_B, accumulator)
+
+        a_ptrs += BLOCK_K_PACKED_A * stride_ak
+        b_ptrs += BLOCK_K_PACKED_B * stride_bk
+
+        a_scale_ptr += BLOCK_K_SCALE
+        b_scale_ptr += BLOCK_K_SCALE
+
+    offs_cm = pid_m * BLOCK_M + ttgl.arange(0, BLOCK_M, layout=ttgl.SliceLayout(1, WMMA_LAYOUT))
+    offs_cn = pid_n * BLOCK_N + ttgl.arange(0, BLOCK_N, layout=ttgl.SliceLayout(0, WMMA_LAYOUT))
+    c_ptrs = c_ptr + stride_cm * offs_cm[:, None] + stride_cn * offs_cn[None, :]
+    c_mask = (offs_cm[:, None] < M) & (offs_cn[None, :] < N)
+    ttgl.store(c_ptrs, accumulator, mask=c_mask)
+
+
+@pytest.mark.parametrize("BLOCK_M, BLOCK_N, BLOCK_K", [(32, 32, 64), (32, 32, 128)])
+@pytest.mark.parametrize("DTYPE_A", ["float8_e5m2", "float8_e4m3", "float4"])
+@pytest.mark.parametrize("DTYPE_B", ["float8_e5m2", "float8_e4m3", "float4"])
+def test_compile_mxgemm(BLOCK_M, BLOCK_N, BLOCK_K, DTYPE_A, DTYPE_B):
+    scale_block = 32
+
+    triton_dtype_converter = {'float8_e5m2': "fp8e5", "float8_e4m3": "fp8e4nv", "float4": "u8"}
+    dot_scaled_dtype_converter = {'float8_e5m2': "e5m2", "float8_e4m3": "e4m3", "float4": "e2m1"}
+
+    k = triton.compile(
+        gluon._runtime.GluonASTSource(
+            fn=mxgemm_kernel, signature={
+                "a_ptr": f"*{triton_dtype_converter[DTYPE_A]}", "b_ptr": f"*{triton_dtype_converter[DTYPE_B]}", "c_ptr":
+                "*fp32", "a_scale": "*u8", "b_scale": "*u8", "M": "i32", "N": "i32", "K": "i32", "stride_am": "i32",
+                "stride_ak": "i32", "stride_bk": "i32", "stride_bn": "i32", "stride_cm": "i32", "stride_cn": "i32",
+                "stride_scale": "i32", "DTYPE_A": "constexpr", "DTYPE_B": "constexpr", "SCALE_BLOCK": "constexpr",
+                "BLOCK_M": "constexpr", "BLOCK_N": "constexpr", "BLOCK_K": "constexpr", "GROUP_SIZE_M": "constexpr"
+            }, constexprs={
+                "DTYPE_A": dot_scaled_dtype_converter[DTYPE_A], "DTYPE_B": dot_scaled_dtype_converter[DTYPE_B],
+                "SCALE_BLOCK": scale_block, "BLOCK_M": BLOCK_M, "BLOCK_N": BLOCK_N, "BLOCK_K": BLOCK_K, "GROUP_SIZE_M":
+                1
+            }), target=GPUTarget("hip", 'gfx1250', 32))
+
+    amdgcn = k.asm["amdgcn"]
+    pattern = "v_wmma_scale_f32_16x16x128_f8f6f4"
+    assert re.search(pattern, amdgcn), f"Can't find instruction {pattern} in AMDGCN assembly"
+
+
+@pytest.mark.parametrize("M, N, K", [(32, 32, 128), (64, 64, 256), (128, 128, 512), (1, 8192, 512), (1024, 1024, 128)])
```

Comment:
`(64, 64, 256)` and `(1024, 1024, 128)` doesn't bring much additional value to test--given single block is covered with `(32, 32, 128)` while multi block case is covered with `(128, 128, 512)`.